docs/container_setup/github_codespace/creating_codespace.md

```diff
@@ -19,3 +19,27 @@ GitHub Codespaces' button and then click the green 'Create Codespace' button.
 
 3. You will be taken to a VSCode editor within your browser.
 ![codespace](../../assets/rdev3.png)
+
+4. Rename your codespace for easier identification:
+
+By default, your codespace will have a randomly generated name. Renaming it to
+something descriptive — such as referencing the issue or feature you’re working
+on makes it much easier to find and manage your codespaces, especially if
+you use several.
+
+To rename your codespace:**
```

Comment:
```suggestion
on - makes it much easier to manage your codespaces:
```

Try to keep the text succinct (so people bother to read it!) and avoid repetition